Jenkinsfile

```diff
@@ -0,0 +1,43 @@
+// Define the test routine for different python versions
+def test_python(pythonVersion)
+{
+  node {
+    // Unstash the source on this node
+    unstash name: 'source'
+    // Set up the environment and test
+    withCredentials([[$class: 'UsernamePasswordMultiBinding', credentialsId: 'clientlibs-test', usernameVariable: 'DB_USER', passwordVariable: 'DB_PASSWORD']]) {
+      try {
+        sh """virtualenv tmp -p /usr/local/lib/python${pythonVersion}/bin/${pythonVersion.startsWith('3') ? "python3" : "python"}
+. ./tmp/bin/activate
+echo \$DB_USER
+export ADMIN_PARTY=true
+export RUN_CLOUDANT_TESTS=1
+export CLOUDANT_ACCOUNT=\$DB_USER
+# Temporarily disable the _db_updates tests pending resolution of case 71610
+export SKIP_DB_UPDATES=1
+pip install -r requirements.txt
+pip install -r test-requirements.txt
+nosetests -w ./tests/unit --with-xunit"""
```

Comment:
Just a tiny nit pick is it possible to get this to line up with `virutalenv` on line 10?